pkg/manifests/manifests.go

```diff
@@ -364,8 +364,10 @@ func (f *Factory) AlertmanagerMain(host string, trustedCABundleCM *v1.ConfigMap)
 
 	if trustedCABundleCM != nil {
 		volumeName := "alertmanager-trusted-ca-bundle"
-		a.Spec.VolumeMounts = append(a.Spec.VolumeMounts, trustedCABundleVolumeMount(volumeName, "/etc/pki/alertmanager-ca-bundle/"))
+		volumePath := "/etc/pki/ca-trust/extracted/pem/"
+		a.Spec.VolumeMounts = append(a.Spec.VolumeMounts, trustedCABundleVolumeMount(volumeName, volumePath))
 		a.Spec.Volumes = append(a.Spec.Volumes, trustedCABundleVolume(trustedCABundleCM.Name, volumeName))
+		a.Spec.Containers[0].VolumeMounts = append(a.Spec.Containers[0].VolumeMounts, trustedCABundleVolumeMount(volumeName, volumePath))
```

Comment:
We add this to just the first container?

Reply:
updated to go through all containers

Reply:
I was looking more into it. We are adding it into spec of alertmanager CR. This one has only 1 additional container - oauth-proxy. So it shouldn't change anything.